db/models.py

```diff
@@ -50,3 +54,80 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    bio = models.TextField(blank=True, null=True)
+
+    def __str__(self) -> str:
+        return self.username
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(default=now)
```

Comment:
Consider whether using `auto_now_add=True` might be more appropriate for the `created_at` field if it is intended to be set automatically when an `Order` instance is created. Using `default=now` allows for manual setting of the field, which might not be desired.